lambdas/db-migration/src/migrations/20200813174231_create_collections_table.ts

```diff
@@ -3,19 +3,27 @@ import * as Knex from 'knex';
 export const up = async (knex: Knex): Promise<void> =>
   knex.schema.createTable('collections', (table) => {
     table.increments('cumulusId').primary();
-    table.string('name').notNullable();
-    table.string('version').notNullable();
-    table.string('sampleFileName').notNullable();
-    table.text('granuleIdValidationRegex').notNullable();
-    table.text('granuleIdExtractionRegex').notNullable();
-    table.jsonb('files').notNullable();
-    table.string('process');
-    table.string('url_path');
-    table.enum('duplicateHandling', ['error', 'replace', 'skip', 'version']);
-    table.boolean('reportToEms');
-    table.boolean('ignoreFilesConfigForDiscovery');
-    table.jsonb('meta');
-    table.jsonb('tags');
+    table.string('name').comment('Collection short_name registered with the CMR').notNullable();
```

Comment:
Do we not want to update all the `string` fields in here to `text`?

Reply:
.... yes.   Something didn't get pushed up, apologies.